basics/02-Vis-Basics/02-Vis-Basics.Rmd

```diff
@@ -49,7 +60,15 @@ The tutorial focuses on three basic skills:
 1. How to add variables to a graph with **aesthetics**
 1. How to make different "types" of graphs with **geoms** 
 
-In this tutorial, we will use the [core tidyverse packages](http://tidyverse.org/), including `ggplot2`. I've already loaded the packages for you, so let's begin! 
+In this tutorial, we will use the [core tidyverse packages](http://tidyverse.org/), including `ggplot2`. I've already loaded the packages for you, so let's begin!
+
+```{r glossary, echo=FALSE}
+# If we decide to include assumed and defined terms, the four lines below would be put in a function.
+assumes <- paste(params$assumes, collapse = ", ")
+defines <- paste(params$defines, collapse = ", ")
+glossary <- paste(c('**Terms**', '', '- **Assumes:**', assumes, '- **Defines:**', defines, ''), collapse = "\n")
+knitr::asis_output(glossary)
```

Comment:
I like this work _a lot_. I think we need to integrate it into a global system that can identify which tutorials a student should visit to learn specific terms. 

My first guess is that the way to do this will be to place the terms into their own YAML field, which we can then scrape the way we scrape tutorial descriptions. I'll reach out to schedule a meeting to see if you have something in mind.

Reply:
:+1: If https://github.com/rstudio-education/glossRy/ takes off the YAML entries will be keys into a shared (global) glossary that can be loaded as a package rather than raw words. The terms will then be pulled from that glossary to ensure consistency and findability, to allow linkage, and to enable internationalization.

Reply:
That sounds like a great system for definition consistency. Thanks for putting it together.

I'm referring to a second need: at some point we will need to recover this ground to make a course _recommender_ (e.g. Need help with **visual space**? Here are the tutorials that cover it.) Making the terms discoverable in a standardized YAML field seems like hooks enough for that system, but this is the best stage to prepare for it if you have any ideas.